### ISSUE TITLE
'can't load package' if outside GOPATH

### ISSUE THREAD

**Issue by rolldever** [Microsoft/vscode-go#908]
```
file: 'file:///Volumes/Duo/Landing/foo/main.go'
severity: 'Error'
message: 'can't load package: package /Volumes/Duo/Landing/foo:
    ↪ import "/Volumes/Duo/Landing/foo": cannot import absolute path'
at: '1,1'
source: ''
```

```
$ tree .
.
└── main.go
```

main.go:

```go
package main

func main() {
}
```

vscode: 1.10.2
vscode-go: 0.6.57

**Comment by elasti-liranm** [comment]
Happens also in gopath.

**Comment by ramya-rao-a** [comment]
Hi,

This is being tracked in #894. Closing this issue as duplicate

There is a workaround explained in https://github.com/Microsoft/vscode-go/issues/894#issuecomment-290610756.

Will be releasing the fix soon.

**Comment by ramya-rao-a** [comment]
I just released an update (0.6.59) yesterday which has the fix
